### PR TITLE
Comments and delivered fluence figure

### PR DESCRIPTION
**Pull request by koosvanamerongen**
Added some comments to the LaTeX-files and included an illustration of
the delivered fluence g(x) (from my thesis).

### CODE REVIEW

File: latex/method.tex [hunk 1]
```diff
@@ -2,18 +2,20 @@ \section{Method}
...
-The general algorithm presented in this paper is agnostic to the choice of trajectory represention.
+The general algorithm presented in this paper is agnostic to the choice of trajectory representation.
 There is a large literature in trajectory optimization studying trajectory representation. \addref{trajectory representation}
 Here we choose to use shape-preserving cubic splines: the \texttt{PCHIP} spline type in Matlab.
 This choice of trajectory has a few useful properties: it is continuous in leaf position and velocity
 and it makes it simple to represent the trajectory using only the position at a few knot points.
 A piecewise linear spline would also be a reasonable choice.
 
 \todo{why cubic $>$ linear?  Would linear be faster?}
+\KvAcomment{If linear spline is indeed faster, I'd go for that. With low-order polynomials that shouldn't harm too much.}
```
Comment (MatthewPeterKelly) [inline latex/method.tex:13]: I think that I tried linear at some point and then moved on to cubic, but I don't remember why. I want to look back at the code to see if it have a good reason. If not, then maybe I will have the code to that instead.

File: latex/method.tex [hunk 2]
```diff
@@ -35,18 +37,21 @@ \subsection{Computing Delivered Fluence}
...
+Across that domain the Chebyshev polynomials range from $-1$ to $1$.
 We assume we have $T=1$ time unit to reproduce the fluence map $f$ by sliding leaves across the field from left to right.
-Let $v_L(t)$ be the velocity of the left leaf at time $t$ and $v_R(t)$ the right leaf velocity. We assume we fix the starting positions
-of both leaves. We may end up assuming we fix the ending positions too. This is not ideal, in fact we would like to optimize over
-starting and ending leaf positions as well, but to make things more concrete, for now starting positions are assumed fixed and given.
+Let $v_L(t)$ be the velocity of the left leaf at time $t$ and $v_R(t)$ the right leaf velocity. 
+We assume we fix the starting positions of both leaves. 
+We may end up assuming we fix the ending positions too. 
+This is not ideal, in fact we would like to optimize over starting and ending leaf positions as well, but to make things more concrete, for now starting positions are assumed fixed and given.
+\KvAcomment{I'm curious about the start/end position search. I'm using 2 methods: one uses restrictive sampling, the other partial enumeration. Both assume unidirectional trajectories though.}
```
Comment (MatthewPeterKelly) [inline latex/method.tex:54]: This section was written by David, and doesn't totally match the Matlab code. Right now I allow both the starting and ending positions to be arbitrary, although we modify either to be fixed without too much work.

**Review by MatthewPeterKelly — APPROVED** [review]
This looks like a great start!

File: latex/method.tex (hunk 2 above)
Comment (koosvanamerongen) [reply latex/method.tex:54]: Okay. We'll also need to decide on a strategy to search the start/end positions space. I'm not sure what David has in mind. We might go for a simple partial enumeration strategy, enumerating several start/ends near the edges of the field. Of course, this is a restriction, but it's defensible. With only one pair of leaves, we can omit half of the combinations for free due to symmetry arguments (reversing both leaf trajectories and dose rate pattern in time results in the same objective value).

**Pull request merged**